packages/fxa-settings/src/pages/Signin/container.tsx

```diff
@@ -464,10 +485,6 @@ const SigninContainer = ({
     [authClient, flowQueryParams, ftlMsgResolver]
   );
 
-  if (oAuthDataError) {
-    return <OAuthDataError error={oAuthDataError} />;
```

Comment:
We are ignoring the oAuthDataError because the data is now validated above with query param validation and any errors will hit the AppErrorBoundary?